### PR TITLE
Feat/modularize

### PR DESCRIPTION
**Pull request by guidiaz**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    return "HTTP-GET"
+  } else if (method == 2) {
+    return "RNG"
+  } else if (method == 3) {
+    return "HTTP-POST"
+  } else if (method == 4) {
+    return "HTTP-HEAD"
+  } else {
+    return method.toString()
+  }
```
Comment (Tommytrg) [inline src/utils.js:99]: What do you think about using a map here?
```suggestion
  if (!method) {
    return "HTTP-GET
  }

  const methodNameMap = {
    0: "UNKNOWN"
    1: "HTTP-GET"
    2: "RNG"
    3: "HTTP-POST"
    4: "HTTP-HEAD"
   } 
  
  const methodName = methodNameMap[method]
  
  return methodName || method.toString()
```

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    return "Integer"
+  } else if (type == 5) {
+    return "Float"
+  } else if (type == 6) {
+    return "Map"
+  } else if (type == 7) {
+    return "String"
+  } else {
+    return "(Undetermined)"
+  }
```
Comment (Tommytrg) [inline src/utils.js:119]: What do you think about using here a map?
```suggestion
  const typeMap = {
    1: "Array"
    2: "Bool"
    3: "Bytes"
    4: "Integer"
    5: "Float"
    6: "Map"
    7: "String"
  }
  
  return typeMap[type] || "(Undetermined)"
```

File: src/utils.js [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+// function stringifyWitnetRequestMethod(method) {
+//   switch (method) {
+//     case Witnet.retrievals.Methods.HttpGet: return "HTTP-GET";
+//     case Witnet.retrievals.Methods.HttpHead: return "HTTP-HEAD";
+//     case Witnet.retrievals.Methods.HttpPost: return "HTTP-POST";
+//     case Witnet.retrievals.Methods.RNG: return "WITNET-RNG";
+//     default: return "UNKNOWN"
+//   }
+// }
```
Comment (Tommytrg) [inline src/utils.js:239]: Could we remove this comment?

**Review by Tommytrg — COMMENTED** [review]
I've added some code suggestions we could discuss in this PR or leave them for a different one if we want to keep this only for the modularization.

File: .gitignore [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Dependency directories
 node_modules
-yarn.lock
+*lock*
```
[the remark this answers is not in the document]
Comment (guidiaz) [reply .gitignore:3]: yarn will be

**Review by Tommytrg — APPROVED** [review]

**Pull request merged**